packages/fungi/src/ContextChips/index.tsx

```diff
@@ -0,0 +1,62 @@
+import styled from "@emotion/styled";
+import { IconButton } from "@leafygreen-ui/icon-button";
+import { RichLink } from "@lg-chat/rich-links";
+import Icon from "@evg-ui/lib/components/Icon";
+import { size } from "@evg-ui/lib/constants/tokens";
+import { ContextChip } from "../Context/context";
+
+export type ContextChipsProps = {
+  chips: ContextChip[];
+  dismissible: boolean;
+  onDismiss?: (chip: ContextChip) => void;
+};
+
+export const ContextChips: React.FC<ContextChipsProps> = ({
+  chips,
+  dismissible,
+  onDismiss,
+}) => (
+  <ChipContainer dismissible={dismissible}>
+    {chips.map((chip) => (
+      <SingleChip
+        key={chip.identifier}
+        data-cy={chip.identifier}
+        data-dismissible={dismissible}
+      >
+        <RichLink
```

Comment:
Did you see this usage example? I think I was able to get it working and it seems preferable

https://github.com/mongodb/leafygreen-ui/tree/main/chat/message#messagelinks